### PR TITLE
[CI:DOCS] Update release notes for v1.8.1

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
Did you update the version of Buildah?

**Comment by mheon** [comment]
I'm going off the vendor PR you made. I know it's pinned to master right now, but this is just an RC - we can cut a buildah release and revendor next week before we cut 1.8.1 final

**Comment by rhatdan** [comment]
Ok since there is a bug in buildah that blocks installation of files with file caps.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## 1.8.1
+### Features
+- Many networking-related flags have been added to `podman pod create` to enable customization of pod networks, including `--add-host`, `--dns`, `--dns-opt`, `--dns-search`, `--ip`, `--mac-address`, `--network`, and `--no-hosts`
+- The `podman ps --format=json` command now includes the ID of the image containers were created with
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: 'with' is generally a joining word and especially odd to end a "sentence" with that doesn't have periods.  Perhaps "image containers" -> "image that the containers"

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+- The `podman create` and `podman run` commands now support the `--device-cgroup-rule` flag ([#4876](https://github.com/containers/libpod/issues/4876))
+- While the HTTP API remains in alpha, many fixes and additions have landed. These are documented in a separate subsection below
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: HTTP or HTTPS?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+
+### Bugfixes
+- Fixed CVE-2020-1726, a security issue where volumes manually populated before first being mounted into a container could have those contents overwritten on first being mounted into a container
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: suggest "on first" -> "when first"

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+- Fixed a bug where the `podman login` and `podman logout` commands required the registry to log into be specified ([#5146](https://github.com/containers/libpod/issues/5146))
+- Fixed a bug where detached rootless Podman containers could not forward ports ([#5167](https://github.com/containers/libpod/issues/5167))
+- Fixed a bug where rootless Podman could fail to run if the pause process had died
+- Fixed a bug where Podman ignored labels that were specified with only a key and no value ([#3854](https://github.com/containers/libpod/issues/3854))
+- Fixed a bug where Podman would fail to create named volumes when the backing filesystem did not support SELinux labelling ([#5200](https://github.com/containers/libpod/issues/5200))
+- Fixed a bug where `--detach-keys=""` would not disable detaching from a container ([#5166](https://github.com/containers/libpod/issues/5166))
+- Fixed a bug where the `podman ps` command was too aggressive when filtering containers and would force `--all` on in too many situations
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: suggest "on in too many" -> "to be on in too many"

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,54 @@
...
+- The `json` endpoint for containers has received many fixes
+- Filtering images and containers has been greatly improved, with many bugs fixed and documentation improved
+- Image creation endpoints (commit, pull, etc) have seen many fixes
+- Server timeout has been fixed so that long operations will no longer trigger the timeout and shut the server down
+- The `stats` endpoint for containers has seen major fixes and now provides accurate output
+- Handling the HTTP 304 status code has been fixed for all endpoints
+- Many fixes have been made to API documentation to ensure it matches the code
+
+### Misc
+- Updated vendored Buildah to v1.14.1
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: This is kinda funky.  There is (as of yet) not Bulidah v1.14.1.  However when Dan did the vendoring it claimed 1.14.1 in Libpod.  We may want to change this to 1.14.0-dev

**Pull request merged**